flutter_vlc_player/README.md

```diff
@@ -0,0 +1,115 @@
+#  VLC Player Plugin
+A VLC-powered alternative to Flutter's video_player that supports iOS and Android.
+
+<p float="left">
+  <img src="https://github.com/solid-software/flutter_vlc_player/blob/master/img_example_v4.jpg?raw=true" height="400">
+  <img src="https://github.com/solid-software/flutter_vlc_player/blob/master/imgpsh_mobile_save.jfif?raw=true" height="400">
+</p>
+
+
+
+## Installation
+
+### iOS
+
+If you're unable to view media loaded from an external source, you should also add the following:
+```xml
+<key>NSAppTransportSecurity</key>
+<dict>
+  <key>NSAllowsArbitraryLoads</key>
+  <true/>
+</dict>
+```
+For more information, or for more granular control over your App Transport Security (ATS) restrictions, you should
+[read Apple's documentation](https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads).
+
+Make sure that following line in `<project root>/ios/Podfile` uncommented:
+
+`platform :ios, '9.0'`
+
+> NOTE: While the Flutter `video_player` is not functional on iOS Simulators, this package (`flutter_vlc_player`) **is**
+> fully functional on iOS simulators.
+
+To enable vlc cast functionality for external displays (chromecast), you should also add the following:
+
+```xml
+<key>NSLocalNetworkUsageDescription</key>
+<string>Used to search for chromecast devices</string>
+<key>NSBonjourServices</key>
+<array>
+  <string>_googlecast._tcp</string>
+</array>
+```
+
+<hr>
+
+### Android
+To load media/subitle from an internet source, your app will need the `INTERNET` permission.  
+This is done by ensuring your `<project root>/android/app/src/main/AndroidManifest.xml` file contains a `uses-permission`
+declaration for `android.permission.INTERNET`:
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+As Flutter includes this permission by default, the permission is likely already declared in the file.
+
+Note that if you got "Cleartext HTTP traffic to * is not permitted"
+you need to add the `android:usesClearTextTraffic="true"` flag in the AndroidManifest.xml file, or define a new "Network Security Configuration" file. For more information, check https://developer.android.com/training/articles/security-config
+
+<br>
+
+In order to load media/subtitle from internal device storage, you should put the storage permissions as follows:
+```xml
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+```
+In some cases you also need to add the `android:requestLegacyExternalStorage="true"` flag to the Application tag in AndroidManifest.xml file.
+
+After that you can access the media/subtitle file by 
+
+    "/storage/emulated/0/{FilePath}"
+    "/sdcard/{FilePath}"
+
+<hr>
+
+## Quick Start
+To start using the plugin, copy this code or follow the example project in 'flutter_vlc_player/example'
+
+```dart
+TBD
```

Comment:
Could you please add a minimal example here?

Reply:
Added!